kedro/framework/session/session.py

```diff
@@ -75,6 +76,12 @@ def _jsonify_cli_context(ctx: click.core.Context) -> Dict[str, Any]:
     }
 
 
+def _clear_hook_manager(hook_manager):
+    name_plugin_pairs = hook_manager.list_name_plugin()
+    for name, plugin in name_plugin_pairs:
+        hook_manager.unregister(name=name, plugin=plugin)  # pragma: no cover
```

Comment:
Do we need this `no cover` bit? I thought this line would be reached on every session close, hence covered?

Reply:
No we don't, good catch!